modules/aerospike/aerospike_test.go

```diff
@@ -0,0 +1,54 @@
+package aerospike_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go"
+	as "github.com/testcontainers/testcontainers-go/modules/aerospike"
+)
+
+// TestAerospikeDB tests the AerospikeDB container functionality
+// It includes tests for starting the container with a valid image,
+// applying container customizations, and handling context cancellation.
+// It also includes a test for an invalid image to ensure proper error handling.
+// The tests use the testcontainers-go library to manage container lifecycle
+func TestAeroSpikeDB(t *testing.T) {
+	t.Run("fails with invalid image", func(t *testing.T) {
```

Comment:
suggestions: let's use subtests name without whitespaces, in order to make it easier to just copy&paste errors and run them in the terminal. More below
```suggestion
	t.Run("invalid-image", func(t *testing.T) {
```